playbooks/openshift-hosted/private/redeploy-router-certificates.yml

```diff
@@ -87,14 +100,13 @@
     - name: Add serving-cert-secret annotation to router service
       command: >
         {{ openshift_client_binary }} annotate service/router
-        service.alpha.openshift.io/serving-cert-secret-name=router-certs
+        service.alpha.openshift.io/serving-cert-secret-name=router-metrics-tls
```

Comment:
I don't think this is necessary, API server should regenerate both so annotation name is correct

Reply:
The initial installation would set up as "router-metrics-tls", it changed to "router-certs" after running the redeploy playbooks, so I just change it back. And router should be providing wildcard certificates against external access, service serving secret is generated internal service certificates for internal communication, it's not correct for router certificates secret.